pkgs/development/python-modules/onnx/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv
+, lib
+, fetchPypi
+, buildPythonPackage
+, protobufc
+, protobuf
+, cmake
+, pytestrunner
+, numpy
+, typing
+, typing-extensions
+, mypy
+}:
+
+buildPythonPackage rec {
+  pname = "onnx";
+  version = "1.2.2";
+
+  # FIXME: mypy and typing-extensions are intended to produce typing stubs.
+  # Sadly, they dont go saying `Failed to generate mypy stubs: No module
+  # named 'google.protobuf.compiler'`
+  nativeBuildInputs = [ cmake protobufc mypy ];
+  propagatedBuildInputs = [ typing typing-extensions ];
+  buildInputs = [ protobuf numpy pytestrunner ];
```

Comment:
test requirements should go in `checkInputs`

Reply:
Done